lib/iris/tests/unit/coords/test__DimensionalMetadata.py

```diff
@@ -0,0 +1,26 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""Unit tests for the :class:`iris.coords._DimensionalMetadata` class."""
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+from iris.coords import _DimensionalMetadata
+
+
+class Test___init____abstractmethod(tests.IrisTest):
+    def test(self):
```

Comment:
I feel like this method ought to have a more descriptive name.

Reply:
Actually, having had a look at other examples in the tests, I can see there is precedent for this.